package.json

```diff
@@ -39,6 +39,7 @@
 		]
 	},
 	"devDependencies": {
+		"@0no-co/graphqlsp": "^1.3.0",
```

Comment:
this actually works pretty well, but isn't used right now so it should be removed